examples/langchain-huggingface/README.md

```diff
@@ -0,0 +1,68 @@
+# Data query example
+
+Example of integration with HuggingFace Inference API with help of [langchaingo](https://github.com/tmc/langchaingo).
+
+## Setup
+
+Download the LocalAI and start the API:
+
+```bash
+# Clone LocalAI
+git clone https://github.com/go-skynet/LocalAI
+
+cd LocalAI/examples/langchain-huggingface
+
+docker-compose up -d
+```
+
+Node: Ensure you've set `HUGGINGFACEHUB_API_TOKEN` environment variable, you can generate it
```

Comment:
This kinds of settings needs more visibility in the docs too. Maybe a good spot for this is: https://localai.io/advanced/index.html

Reply:
Also done